.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,41 @@
+name: Bug report
+description: |
+  Report a bug. For security vulnerabilities see Report a security vulnerability in the templates.
+title: "BUG:"
+labels: [00 - Bug]
+
+body:
+- type: textarea
+  attributes: 
+    label: "Describe the issue:"
+  validations:
+    required: true
+
+- type: textarea
+  attributes:
+    label: "Reproduce the code example:"
+    description: |
+      A short code example that reproduces the problem/missing feature. It should be self-contained, i.e., possible to run as-is via
+      'python myproblem.py'.
+    placeholder: |
+      ```python
+      import numpy as np
+      << your code here >>
+      ```
+    render: markdown
+  validations:
+    required: true
+  
+- type: textarea
+  attributes:
+    label: "Error message:"
+    description: "Please include full error message, if any (starting from line Traceback: ...). If you are reporting a segfault please include a GDB traceback, which you can generate by following https://github.com/numpy/numpy/blob/main/doc/source/dev/development_environment.rst#debugging."
+  validations:
+    required: false  
+
+- type: textarea
+  attributes:
+    label: "NumPy/Python version information:"
+    description: Output from 'import sys, numpy; print(numpy.__version__, sys.version)'
```

I think markdown renders here?
```suggestion
    description: Output from `import sys, numpy; print(numpy.__version__, sys.version)`
```

---

Yes! I was not aware; I will edit it.